tests/e2e/fixtures.ts

```
@@ -86,48 +94,76 @@ export const test = base.extend<Fixtures, WorkerFixtures>({
+            } catch (error) {
+              if (
+                error instanceof Prisma.PrismaClientKnownRequestError &&
+                error.code === "P2003" &&
+                attempt < maxAttempts - 1
+              ) {
+                await new Promise((resolve) =>
+                  setTimeout(resolve, 100 * (attempt + 1))
+                );
+                continue;
+              }
+              throw error;
```

Biome (2.1.2)

[error] Unsafe usage of 'throw'.

'throw' in 'finally' overwrites the control flow statements inside 'try' and 'catch'.

(lint/correctness/noUnsafeFinally)